sbt-plugin/src/sbt-test/sbt-akkaserverless/basic/test

```diff
@@ -1,13 +1,73 @@
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/action3/some_action/SomeActionProvider.scala
+
+# com/example/action4/some_other_action_api.proto
+$ exists src/main/scala/com/example/action4/some_other_action_api/SomeOtherActionImpl.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/action4/some_other_action_api/AbstractSomeOtherAction.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/action4/some_other_action_api/SomeOtherActionHandler.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/action4/some_other_action_api/SomeOtherActionProvider.scala
+
+# com/example/action5/yet_another_action.proto
+$ exists src/main/scala/com/example/action5/yet_another_action/YetAnotherActionImpl.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/action5/yet_another_action/AbstractYetAnotherAction.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/action5/yet_another_action/YetAnotherActionHandler.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/action5/yet_another_action/YetAnotherActionProvider.scala
+
+# com/example/view/user_view.proto
+$ exists src/main/scala/com/example/view/user_view/UserByNameView.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/view/user_view/AbstractUserByNameView.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/view/user_view/UserByNameViewHandler.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/view/user_view/UserByNameViewProvider.scala
+$ exists src/main/scala/com/example/view/user_view/AdditionalViewImpl.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/view/user_view/AbstractAdditionalView.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/view/user_view/AdditionalViewProvider.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/view/user_view/AdditionalViewHandler.scala
+
+# com/example/valueentity/some_value_entity_api.proto
+# com/example/valueentity/domain/some_value_entity_domain.proto
+$ exists src/main/scala/com/example/valueentity/domain/some_value_entity_domain/SomeValueEntity.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/valueentity/domain/some_value_entity_domain/AbstractSomeValueEntity.scala
+
+# com/example/valueentity/user_api.proto
+# com/example/valueentity/domain/user_domain.proto
+$ exists src/main/scala/com/example/valueentity/domain/user_domain/User.scala
+$ exists target/scala-2.13/src_managed/main/akkaserverless/com/example/valueentity/domain/user_domain/AbstractUser.scala
 
 > Test/compile
-$ exists target/scala-2.13/src_managed/test/akkaserverless/foo/bar/BazSpec.scala
-$ absent target/scala-2.13/src_managed/test/akkaserverless/foo/bar/Baz.scala
```

Comment:
I didn't find it useful to keep testing this. Very soon we will have the real stuff here.